tutorials/CMakeLists.txt

```diff
@@ -63,6 +63,7 @@ if(NOT extraAieCcFlags)
   endif()
 else()
   set(LibXAIE_ROOT ${CMAKE_INSTALL_PREFIX}/runtime_lib/xaiengine)
+  set(XILINX_XAIE_LIBS ${CMAKE_INSTALL_PREFIX}/runtime_lib/xaiengine)
```

Comment:
Are the tutorials run with 'ninja check-aie'?

Reply:
tutorials run with 'ninja check-tutorials'